scripts/localnet-blocks-test.sh

```diff
@@ -49,7 +49,7 @@ while [ ${CNT} -lt $ITER ]; do
     docker restart ${rand_container} &>/dev/null &
   fi
 
-  let CNT=CNT+1
+  CNT=$((CNT + 1))
```

Comment:
no drive by PRs can fix code that isn't actually broken. if this is broken open an issue to explain why